### ISSUE TITLE
Symbol's funciton defination is void: dump-jump-xref-activate

### ISSUE THREAD

**Issue by sachyya** [jacktasia/dumb-jump#364]
Hello,

New to emacs and trying to make it just work.

I just installed dump-jump from melp and using it for PHP funcitons and configured it as shown in docs in my `.emacs` file like below:

`(add-hook 'xref-backend-functions #'dumb-jump-xref-activate)`

But when using M-. the message is shown as below

`Symbol's funciton defination is void: dump-jump-xref-activate`

**Comment by jacktasia** [comment]
Looks like you have "dump" and not "dumb" somewhere.

**Comment by sachyya** [comment]
I removed the package and reinstalled it now the minibuffer shows:
`Symbol's funciton defination is void: dumb-jump-xref-activate`

Do i need more configuration?

**Comment by jacktasia** [comment]
> Do i need more configuration?

Can you share _exactly_ what you have in your config right now. You have both `function` and `definition` misspelled in your quote, so it seems like something is off here, and that makes it harder to help if we're not getting the exact information for the issue.

**Comment by wildstyl3r** [comment]
Hello!
I had the same problem, looked up the file of package and found that it appears because the current state of main branch was not released yet The latest version in MELPA is 0.5.3, which is the latest release, and it just had not this function that time. Looks like we (the users) need new release / version bump :)

**Comment by phikal** [comment]
> UPD: Okay, I'm still too new to emacs, just found that in melpa one can get not only stable versions )

True, but a new release would be good either way (also relating to #399, since NonGNU ELPA is stable by default).

**Comment by JaapJoris** [comment]
Obligatory "me too" comment, using Debian.

First, I tried `M-x package-install` but got the error:

    Install package: dumb-jump [No match]

Then, I ran `sudo apt install elpa-dumb-jump` which installed version 0.5.3-1 and added `(add-hook 'xref-backend-functions #'dumb-jump-xref-activate)` to my `.emacs` file. This is the exact error I got:

    Symbol's function definition is void: dumb-jump-xref-activate

(I double checked for spelling errors)

**Comment by phikal** [comment]
Jaap Joris Vens ***@***.***> writes:

> Obligatory "me too" comment, using Debian.
>
> First, I tried `M-x package-install` but got the error:
>
>     Install package: dumb-jump [No match]

Do you have MELPA configured and have you refreshed the package index?

> Then, I ran `sudo apt install elpa-dumb-jump` which installed version
> 0.5.3-1 and added `(add-hook 'xref-backend-functions

If the version if 0.5.3, then there will be no xref support.  As such,
it is not a surprise to see that the symbol is void.

> #'dumb-jump-xref-activate)` to my `.emacs` file. This is the exact
> error I got:
>
>     Symbol's function definition is void: dumb-jump-xref-activate
>
> (I double checked for spelling errors)

-- 
	Philip Kaludercic

**Comment by JaapJoris** [comment]
> Do you have MELPA configured and have you refreshed the package index?

Apparently, I have not, and by falling back to `apt install elpa-dumb-jump` I installed an older version which actually caused this problem for me. Using an older version is presumably also the cause of OP's problem, or perhaps the misspelling of dumb-jump as dum*p*-jump (which I've been guilty of in the past as well).

These are both in no way the fault of dumb-jump, so I'm sorry for bothering you with it. The solution would be for the Debian Emacsen team to update https://salsa.debian.org/emacsen-team/dumb-jump-el

Thank you very much for your quick response, and feel free to close this issue.
